manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml

```diff
@@ -39,6 +39,14 @@ spec:
           for: 15m
           labels:
             severity: critical
+        - alert: PodsUnableToDelete
+          annotations:
+            summary: Some pods are unable to be deleted.
+            description: Some pods have been unable to be deleted for last 24h (since their grace period for termination has been reached).
```

Comment:
We don't actually have access to their termination grace period, do we?  These are pods that were initially asked to delete 24h ago.  I expect in most cases, the termination grace period will be smaller than 24h.  But maybe not?  If we can find a way to start the timer when they hit their termination grace period, that would be great.

Reply:
actually the grace period is embedded in the deletionTimestamp, so basically the deletion timestamp is in the future. And after that there is probably something wrong going on. So I guess we could get with even much smaller window than 24h